packages/core/src/components/toast/toaster.tsx

```diff
@@ -169,6 +189,15 @@ export class Toaster extends AbstractPureComponent<IToasterProps, IToasterState>
         }
     }
 
+    private hasToastWithKey(key: string) {
+        for (const toast of this.state.toasts) {
+            if (toast.key === key) {
+                return true;
+            }
+        }
+        return false;
+    }
+
```

Comment:
since `Array.find()` doesn't seem to be available.

Reply:
could do it with a `filter().length > 0`